backend/tests/backend/db/test_database.py

```
@@ -0,0 +1,48 @@
+    SQLModel.metadata.drop_all(engine)
+    engine.dispose()
+
+@pytest.fixture(name="session")
+def session_fixture(engine):
+    with Session(engine) as session:
+        yield session
+
+def test_save_and_get_file(session: Session):
+    # Create a test file record
+    file_id = "test-file-id"
+    test_file = save_file(
```

Ruff (0.8.0)

Local variable `test_file` is assigned to but never used

Remove assignment to unused variable `test_file`

(F841)